packages/ilmomasiina-models/src/schema/event/attributes.ts

```diff
@@ -4,12 +4,18 @@ import { Nullable } from "../utils";
 
 export const eventSlug = Type.String({
   description: "Event slug, used for accessing the event by URL.",
+  minLength: 1,
```

Comment:
Should the slug be always longer than one character maybe say min. 4

Reply:
Might be desirable, but then users might be unable to save events they've created previously without changing the URL. So we might need to trust the (admin) users here.